sys/net/application_layer/gcoap/forward_proxy.c

```diff
@@ -443,11 +359,11 @@ static int _gcoap_forward_proxy_via_coap(coap_pkt_t *client_pkt,
     pkt.hdr->id = client_pkt->hdr->id;
 
     if (token_len) {
-        memcpy(coap_get_token(&pkt), coap_get_token(client_pkt), token_len);
+        memcpy(pkt.token, client_pkt->token, token_len);
```

Comment:
note: #17983 will remove the `token` struct member.

Reply:
The change seems unrelated anyway. Probably a rebase error.

Reply:
Reverted in 3f718f6